addons/html_builder/static/src/plugins/image_gallery_option_plugin.js

```diff
@@ -401,6 +403,29 @@ class ImageGalleryOption extends Plugin {
         );
     }
 
+    onRemove(elementToRemove) {
+        // If the removed element is an image from a gallery, set the flag to true
+        // and store the gallery element for afterRemove
+        if (!this.isGalleryImage) {
+            this.isGalleryImage = false;
+        }
```

Comment:
I don't understand this condition. If `this.isGalleryImage` is false then we set it to false ?

Reply:
It's to initialize the value as false if the the variable doesn't exit, but yeah we actually don't need the variable and can use the gallery element we are editing. Thanks for the checking! :smile_cat: